### DISCUSSION TITLE
react-i18next Package Not Available in Maven Repository

### DISCUSSION THREAD

**Discussion by sotobotero** [mvnpm/mvnpm#9727] [General]
Hi Guys:
I am encountering an issue while trying to add the react-i18next dependency to my project. 
The specific error message I receive is:

Could not find artifact org.mvnpm:react-i18next:jar:15.0.3 in central (https://repo.maven.apache.org/maven2)

It seems that the package is not available in the Maven Central repository, leading to this resolution failure. I have tried to force updates and clear the local cache, even I have tried with different versions  but the issue persists.

Could you please investigate this matter? Thank you!

 <dependency>
            <groupId>org.mvnpm</groupId>
            <artifactId>react-i18next</artifactId>
            <version>15.0.3</version>
            <scope>provided</scope>
        </dependency>

 <dependency>
            <groupId>org.mvnpm</groupId>
            <artifactId>react-i18next</artifactId>
            <version>9.0.10</version>
            <scope>provided</scope>
        </dependency>

**Comment by phillip-kruger** [comment]
Hi @sotobotero 

If you want to have access to all npm package, you need to add the mvnpm repo to your setting file:

```
<settings>
    <profiles>
        <profile>
            <id>mvnpm-repo</id>
            <repositories>
                <repository>
                    <id>central</id>
                    <name>central</name>
                    <url>https://repo.maven.apache.org/maven2</url>
                </repository>
                <repository>
                    <snapshots>
                        <enabled>false</enabled>
                    </snapshots>
                    <id>mvnpm.org</id> 
                    <name>mvnpm</name>
                    <url>https://repo.mvnpm.org/maven2</url>
                </repository>
            </repositories>
        </profile>
    </profiles>

    <activeProfiles>
        <activeProfile>mvnpm-repo</activeProfile>
    </activeProfiles>

</settings>
```
See https://mvnpm.org/doc

This way, the react-i18next and all it's transitive dependencies will be available, and one you download them once from mvnpm repo, it will also be synced (and kept in sync) with maven central.

Let me know if this works.

**Comment by sotobotero** [comment]
Thank you so much, it Works.
